src/extractor/guidance/turn_lane_handler.cpp

```diff
@@ -701,15 +700,12 @@ Intersection TurnLaneHandler::simpleMatchTuplesToTurns(Intersection intersection
         std::move(intersection), lane_data, node_based_graph, lane_description_id, id_map);
 }
 
-Intersection
-TurnLaneHandler::handleSliproadTurn(Intersection intersection,
-                                    const LaneDescriptionID lane_description_id,
-                                    LaneDataVector lane_data,
-                                    const Intersection &previous_intersection,
-                                    const LaneDescriptionID &previous_lane_description_id,
-                                    const LaneDataVector &previous_lane_data)
+Intersection TurnLaneHandler::handleSliproadTurn(Intersection intersection,
+                                                 const LaneDescriptionID lane_description_id,
+                                                 LaneDataVector lane_data,
+                                                 const Intersection &previous_intersection)
 {
-    const auto sliproad_index =
+    const std::size_t sliproad_index =
```

Comment:
Was this change needed or is it only for readability? (I'm fine with both, just wondering)